skafka/src/test/scala/com/evolutiongaming/skafka/consumer/RebalanceListener1Spec.scala

```diff
@@ -85,8 +76,10 @@ object RebalanceListener1Spec {
     // one way is to import all methods, and then do `seek(...)`/`position(...)`/etc
     import RebalanceCallback._
 
+    // TODO update example in docs when such way of usage is finalized in PR
+    val consumer = RebalanceCallback.api[F]
```

Comment:
JFYI: I'm personally not against having such api, but don't see a lot of value in it.

Reply:
I'm thinking about updating example with extending `RebalanceListener1WithConsumer` so that `consumer` would be already declared and in scope, then no need to write `val consumer = RebalanceCallback.api[F]` in user code, but at the same time we give users a choice how to use the api.
The main reason to have `RebalanceCallback.api[F]` is to fix `F` effect type, this allows better type inference ([example](https://github.com/evolution-gaming/skafka/blob/764b3716a35636a5f93da65177674710f7bb4435/skafka/src/test/scala/com/evolutiongaming/skafka/consumer/RebalanceListener1SyntaxSpec.scala#L54))